client/cli/src/params/network_params.rs

```diff
@@ -146,6 +146,10 @@ pub struct NetworkParams {
 		verbatim_doc_comment
 	)]
 	pub sync: SyncMode,
+
+	/// Maximum number of blocks per request.
+	#[arg(long, value_name = "COUNT", default_value_t = 64)]
+	pub max_blocks_per_request: u32,
```

Comment:
I'm not convinced that we need a cli option for this? It should be enough for people to be able to configure this in their node implementations.

Reply:
I disagree. The reason for making this a CLI option is that based on what network conditions you have, it may be in your best interest to decrease the number if you're seeing a lot of `NetworkTimeout` messages in the logs. These network conditions can be transient so having to recompile the node every time it needs to be adjusted (I've done it a multiple times over the past week) is quite painful.

I can revert this if you think it's a bad idea.

Reply:
It was discussed back in https://github.com/paritytech/substrate/issues/10130 that it is a good idea to make this configurable. CC @bkchr

Reply:
I mean I'm not against having it configurable in general and if that also is useful for "operators", fine. However then we should update the docs to mention this.

My previous point was that in the issue is was about Subspace that in general has bigger blocks and would need to run with a much smaller default value to support syncing in normal cases. But yeah, fine to keep it as CLI as well as I said above!